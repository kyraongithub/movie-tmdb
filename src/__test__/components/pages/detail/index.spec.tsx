import { render } from '@testing-library/react';
import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import { MemoryRouter } from 'react-router-dom';
import DetailPage from '../../../../components/pages/detail';

describe('Detail', () => {
  it('renders Detail component', () => {
    const queryClient = new QueryClient();
    const { container } = render(
      <QueryClientProvider client={queryClient}>
        <MemoryRouter initialEntries={['/detail/123']}>
          <DetailPage />
        </MemoryRouter>
      </QueryClientProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
