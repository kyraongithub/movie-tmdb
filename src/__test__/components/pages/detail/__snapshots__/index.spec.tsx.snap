// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Detail > renders Detail component 1`] = `
<div>
  <div
    class="flex justify-center items-center min-h-screen bg-gray-900"
  >
    <div
      class="_skeleton_8f912d"
      style="width: 200px; height: 300px; border-radius: 0px;"
    />
  </div>
</div>
`;
